datastore/additional_data/models.py

```diff
@@ -92,3 +92,22 @@ class NSPL(models.Model):
 class CodeName(models.Model):
     code = models.CharField(max_length=9)
     data = JSONField()
+
+
+class GeoLookup(models.Model):
+
+    LSOA = 'lsoa'
+    MSOA = 'msoa'
+    LA = 'la'
+    WARD = 'ward'
+
+    AREA_TYPE = [
+        (LSOA, 'Lower Super Output Area'),
+        (MSOA, 'Middle Super Output Area'),
+        (LA, 'Local Authority'),
+        (WARD, 'Ward'),
+    ]
+
+    areacode = models.CharField(max_length=200, unique=True)
+    areatype = models.CharField(max_length=20, choices=AREA_TYPE)
```

Comment:
As I found out earlier, not to underestimate how much faster indexing on postgres can make things.

Think we could add some indexes on areacode and areatype 
https://docs.djangoproject.com/en/3.0/ref/contrib/postgres/indexes/

https://github.com/ThreeSixtyGiving/datastore/pull/40/commits/73b2f02eaecc3a94bd9f4bc7b12b320503250e48#diff-592aa937943d3ac113983eb645744966

Reply:
good idea - indexes now added